src/desktop/config/migrations/DesktopConfigMigrator.js

```diff
@@ -15,6 +15,8 @@ export default function applyMigrations(migrationFunction: "migrateClient" | "mi
 			oldConfig = applyMigration(require('./migration-0000')[migrationFunction], oldConfig)
 		// no break, fallthrough applies all migrations in sequence
 		case 0:
+			oldConfig = applyMigration(require('./migration-0001')[migrationFunction], oldConfig)
```

Comment:
You added it for version 0, will existing clients get it?

Reply:
That migration is from version 0 to version 1, older clients will either have version 0 and apply it or have no version (pre-0) and upgrade to 0 before applying it.

Reply:
Ah, okay, I didn't notice that you've added 1 now, sorry